src/common/sse.h

```diff
@@ -0,0 +1,127 @@
+/*
+    This file is part of darktable,
+    copyright (c) 2015 Bruce Guenter
```

Comment:
copyright looks messed up

Reply:
Why is it messed up? The PR is that old ...